CMakeLists.txt

```diff
@@ -258,6 +258,8 @@ endif ()
 #  Build the tools
 
 add_subdirectory(docs/man)
+add_subdirectory(mqtt)
+add_subdirectory(mqtt_async)
```

Comment:
These two directories do not exist...

Reply:
These two subdirectories are demos, should not be added to CMakeLists.txt.